stable/elasticsearch/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: elasticsearch
 home: https://www.elastic.co/products/elasticsearch
-version: 1.28.5
+version: 1.28.6
```

Comment:
This is a new feature: we need to bump minor version instead of patch.

Reply:
done :)